examples/cf_version_after.cf

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2020 Northern.tech AS
+
+#  This file is part of Cfengine 3 - written and maintained by Northern.tech AS.
+
+#  This program is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License as published by the
+#  Free Software Foundation; version 3.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+# To the extent this program is licensed as part of the Enterprise
+# versions of Cfengine, the applicable Commercial Open Source License
+# (COSL) may apply to this file if you as a licensee so wish it. See
+# included file COSL.txt.
+
+#+begin_src cfengine3
+body common control
+{
+      bundlesequence => { "test" };
+}
+
+bundle agent test
+{
+  classes:
+    "newer" expression => cf_version_after("3.15");
+
+  reports:
+    newer::
+      "Using new features";
+    !newer::
```

Comment:
Using `!` like this, with a class defined in the same bundle is a bad idea;

https://docs.cfengine.com/docs/master/reference-language-concepts-classes.html#negative-knowledge

(You don't know why it is not defined, or if it is not defined yet but will be later).

I would rewrite the example to something like this:

```
bundle agent main
{
  reports:
    "This would be skipped on older versions"
      if => cf_version_after("3.15");
    "This would be skipped on newer versions"
      unless => cf_version_after("3.15");
}
```